### ISSUE TITLE
Optimise SVG (WIP)

### ISSUE THREAD

**Issue by kud** [Hiswe/gulp-svg-symbols#6]
We need to optimise `svg-symbols.svg`. It shouldn't have `<desc>` tag for SEO context.

(bad) example:

![image](https://cloud.githubusercontent.com/assets/655838/3624422/afb01ff8-0e5c-11e4-9c3c-3d42637a9fd5.png)

Just a memo for a future PR.

```javascript
var gulp       = require('gulp')

var through2   = require('through2')

var plumber    = require('gulp-plumber')
var svgSymbols = require('gulp-svg-symbols')
var filter     = require('gulp-filter')
var imagemin   = require('gulp-imagemin')

module.exports = function() {

  var cssFilter = filter('**/*.css')
  var svgFilter = filter('**/*.svg')

  return gulp.src('src/frontend/glyphicons/**/*.svg')
    .pipe( plumber() )
    .pipe( through2.obj(function ( file, enc, cb ) {

      var fileString = file.contents.toString()

      var titleRegex = /<title>.*<\/title>/gi
      var descRegex = /<desc>.*<\/desc>/gi
      var commentRegex = /<!--.*-->/gi
      var defsRegex = /<defs>.*<\/defs>/gi
      var mSShapeGroupRegex = / +sketch:type=\"MSShapeGroup\"/gi
      var mSPageRegex = / +sketch:type=\"MSPage\"/gi

      fileString = fileString.replace(titleRegex, '')
      fileString = fileString.replace(descRegex, '')
      fileString = fileString.replace(commentRegex, '')
      fileString = fileString.replace(defsRegex, '')
      fileString = fileString.replace(mSShapeGroupRegex, '')
      fileString = fileString.replace(mSPageRegex, '')

      file.contents = new Buffer( fileString )

        this.push( file )
        cb()

    }) )
    .pipe(
      svgSymbols({
        svgId: 'pmd-Svg--%f',
        className: '.pmd-Svg--%f',
        fontSize: 32,
        css: true
      })
    )
    .pipe( cssFilter )
    .pipe( gulp.dest('build') )
    .pipe( cssFilter.restore() )
    .pipe( svgFilter )
    .pipe( gulp.dest('web/assets/images') )

}
```

We also should work on `dest` folder for svg and css. It should be an option, not a filter like I did.

**Comment by Hiswe** [comment]
@kud I have removed all SVG optimisation from this plugin: this should be handled outside gulp-svg-symbols.
I've make some tests with [gulp-cheerio](https://www.npmjs.com/package/gulp-cheerio) on [this repo](https://github.com/Hiswe/gulp-cheerio-clean-svg) it should handle what you were trying to do.

@VinSpee As mentioned above, I've removed SVGO and the optimisations that come along. 
Plugin still maintained ;)
